src/core/Akka.NodeTestRunner/Sink.cs

```diff
@@ -84,7 +88,7 @@ public void Dispose()
     /// that this class produces, so do not remove or refactor it.
     /// </remarks>
     /// </summary>
-    public class SpecPass
+    public class SpecPass : MarshalByRefObject
```

Comment:
This is because XUnit runs the tests in a separate AppDomain and we have to marshal the object to the runner.